source/includes/steps-atlas-cli-deploy-local-prereqs.rst

```diff
@@ -0,0 +1,58 @@
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the dependencies.
+
+      a. Install the :ref:`{+atlas-cli+} <install-atlas-cli>`, 
+         :dbtools:`MongoDB Database Tools </installation/installation/>`,
```

Comment:
Afaik `mongosh` is optional when you decide to use `compass` instead (both are optional).

Reply:
Maybe this could be a section like `compass` section

Reply:
Awesome, thanks @jeroenvervaeke and @fmenezes, made those updates and this should be ready for another look. If this looks good, I'll also apply this to our vector search documentation separately.